src/guest/guest.controller.ts

```
@@ -0,0 +1,564 @@
+    if (!guestSessionId) {
+      return {
+        canAccess: false,
+        reason: 'Missing guest session',
+        storiesRead: 0,
+        remaining: 0,
+        totalAllowed: GUEST_STORY_LIMIT,
+        alreadyRead: false,
+      };
+    }
+
+    const session = await this.guestSessionService.getGuestSession(guestSessionId);
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)